languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/grammar.xml

```diff
@@ -11906,9 +11881,198 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA.
                 <example>Torna-se necessário obedecer a normas de conduta.</example>
                 <example>A Literatura obedece a leis inflexíveis: a da herança, a do meio, a do momento.</example>
             </rule>
-        </rulegroup>
+            <!-- Moved from the PHRASAL_VERBS_ERRORS rulegroup above and split to handle contractions. -->
+            <rule default="off">
```

Comment:
Please change to `<rule default="temp_off">` for all instances. This tag allows us to let the rule run over a big corpus internally to find additional false positives, before we publish it for all users.
`default="off"` completely deactivates the rule.

Reply:
Oops, my bad! Fixed.